### ISSUE TITLE
Migration path From Kodi to SPMC - Importing Add-ons from existing Kodi install to new SPMC install?

### ISSUE THREAD

**Issue by jbl0ggs** [koying/SPMC#477]
Is there a easy migration path from Kodi to SPMC? A method/feature/option to import all/selected add-ons from currently installed Kodi setup to SPMC?

**Comment by ruppert10** [comment]
Hi read this all the way thru it works 100% if you don't skip any steps 

https://github.com/koying/SPMC/issues/88

**Comment by jbl0ggs** [comment]
Thanks! But it seems that discussion is for upgrading from older version SPMC to newer version of SPMC. Would those (4 steps) work for migrating from Kodi to SPMC?

Thanks for the help!

**Comment by ruppert10** [comment]
Yes I did this to transfer all my kodi add-ons and settings I was afraid to update to spmc from kodi thinking I'd have to start over from scratch but this transfers everything all kodi add-ons settings you will not loose anything just start spec one time close it then do all the steps I use especially file

**Comment by ruppert10** [comment]
Oops es file exp to transfer everything over sorry I hit post by mistake good luck

**Comment by ruppert10** [comment]
1. Install 16.1.1
1.  Run 16.1.1 once
2. Delete all (!!!) files incl. folders out of folder /sdcard/Android/data/com.semperpax.spmc16/files/.spmc

4.copy all (!!!) files incl. folders from /sdcard/Android/data/com.semperpax.spmc/files/.spmc to

( in your case it's the Kodi folder you copy from to  ) 

/sdcard/Android/data/com.semperpax.spmc16/files/.spmc good luck

**Comment by ruppert10** [comment]
Last post I mean new spmc version 16.4.2 not the old 16.1.1 version you may as well use the newest version out

**Comment by jbl0ggs** [comment]
Thanks so much for all the info !!!

**Comment by aml9999** [comment]
I realize this is an old thread but it seemed like the right place to ask...

I was using SPMC for a while, when Kodi 17 came out on the shield, i thought i'd give it a try.  I got everything set up in there the way I like it, but I miss all the SPMC features and don't see any advantages to Kodi (or 17, which is still pretty glitchy at 17.1).

I followed the above instructions but am wondering if there's backwards compatability issues.  I noticed that now on SPMC the RSS feed was enabled on the home screen.  I also cant get any of my addons to play and am not getting the dialog to select a source to stream.  General navigation is pretty slow too.

I looked around but am wondering if there's any info out there on settings, files that need to be edited/removed/replaced.  Or do you think something else is going on?

Thanks.  Love SPMC and wish I didn't make that switch in January.
